test/setup/setup.ts

```diff
@@ -3,57 +3,98 @@ import env from "../../src/config/env";
 import "./matchers/to-have-sent-metrics";
 import "./matchers/nock";
 import "./matchers/to-promise";
-import statsd from "../../src/config/statsd";
 import { sequelize } from "../../src/models/sequelize";
-import InstallationTokenCache from "../../src/github/client/installation-token-cache";
-import AppTokenHolder from "../../src/github/client/app-token-holder";
+import { mocked } from "ts-jest/utils";
+// WARNING: Be very careful what you import here as it might affect test
+// in other others because of dependency tree.  Keep imports to a minimum.
```

Comment:
`in other others` hahaha. Not sure what the word was meant to be here

Reply:
It is a strong no go. We shuldn't skip working tests

Reply:
We went through this once before and caused an incident

Reply:
:|